### ISSUE TITLE
Modal: Focus traps when using Popovers inside of Modal content

### ISSUE THREAD

**Issue by EdwardCoyle** [infor-design/enterprise#3644]
**Describe the Problem:**
There are currently focus traps present in the Modal related to Popovers and Tooltips with focusable content. When an open Popover is present inside a Modal, the Popover's focusable elements, like buttons, links, etc, are not accessible via keyboard once focus leaves the Popover and returns to the Modal.

**Steps to Reproduce:**
- Open https://4260-enterprise.demo.design.infor.com/components/modal/test-popover.html
- Open the Modal
- Open the Popover
- Focus the hyperlink/button/etc inside the Popover and use the keyboard to tab around.  Focus will eventually go back to the Modal, but can never escape the Modal to go back to the open Popover.

Focus can be re-established inside the Popover by clicking, but not by keyboard.

**Expected Behavior**
When a popover is open inside of a Modal, the Popovers' focusable contents should become part of the Modal's focusable contents, and allow keyboard access to those elements in the proper tabindex order.

**Additional Context:**
_Originally posted by @EdwardCoyle in https://github.com/infor-design/enterprise/issues/2287#issuecomment-601351922_

**Comment by jbrcna** [comment]
**QA Passed**
v4.30.0-dev
https://master-enterprise.demo.design.infor.com/components/modal/test-external-components?layout=nofrills
